roachtest: activerecord failed

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4467588&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4467588&tab=artifacts#/activerecord) on master @ [eee8ef080a95bc59eb3ddc2bebf623eb78e78db9](https://github.com/cockroachdb/cockroach/commits/eee8ef080a95bc59eb3ddc2bebf623eb78e78db9):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:190,orm_helpers.go:116,activerecord.go:251,test_runner.go:779: 
		Tests run on Cockroach v22.1.0-alpha.1-1740-geee8ef080a
		Tests run against activerecord 6.1
		6640 Total Tests Run
		6639 tests passed
		1 test failed
		19 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_interval_type (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_1) is available in the artifacts' activerecord log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #69080 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/activerecord.go
+++ b/pkg/cmd/roachtest/tests/activerecord.go
@@ -28,7 +28,7 @@ import (
 var activerecordResultRegex = regexp.MustCompile(`^(?P<test>[^\s]+#[^\s]+) = (?P<timing>\d+\.\d+ s) = (?P<result>.)$`)
 var railsReleaseTagRegex = regexp.MustCompile(`^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<point>\d+)\.?(?P<subpoint>\d*)$`)
 var supportedRailsVersion = "6.1"
-var activerecordAdapterVersion = "v6.1.5"
+var activerecordAdapterVersion = "v6.1.6"
 
 // This test runs activerecord's full test suite against a single cockroach node.
 

